tests/integration/cloud-hypervisor-rs/gha-run.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+kata_tarball_dir="${2:-kata-artifacts}"
+gral_dir="$(dirname "$(readlink -f "$0")")"
+source "${gral_dir}/../../common.bash"
+
+declare -r cloud_hypervisor_rs_dir="${gral_dir}/../../../src/runtime-rs"
+declare -r stability_dir="${gral_dir}/../../stability"
+
+function install_cloud_hypervisor_rs() {
+	info "Installing the cloud hypervisor runtime-rs"
+	pushd "${cloud_hypervisor_rs_dir}"
+	HYPERVISOR=cloud-hypervisor make
+	sudo make install
+	popd
+}
+
+function setup_cloud_hypervisor_rs() {
+	info "Verify that the shimv2 runtime-rs is installed"
+	/usr/local/bin/containerd-shim-kata-v2 --version | grep -i rust
+
+	info "Use the cloud hypervisor runtime-rs configuration"
+	sudo ln -sf /usr/shared/defaults/kata-containers/cloud-hypervisor-configuration.toml /usr/shared/defaults/kata-containers/configuration.toml
```

Comment:
Typo:

```suggestion
	sudo ln -sf /usr/share/defaults/kata-containers/cloud-hypervisor-configuration.toml /usr/share/defaults/kata-containers/configuration.toml
```

Also, note to us all that we'll need to remember to change this later if/when #8405, #8306 make the runtime-rs changes.